src/product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,32 @@
+import products from '../data.json';
```

Comment:
To make it more clear, lets separate handlers by subfolders

`src/handlers/getProduct/handler.ts`
`src/handlers/getProduct/handler.test.ts`

`src/handlers/getProducts/handler.ts`
`src/handlers/getProducts/handler.test.ts`

You can create index.ts files in these folders as well for easy export

Play pay attention to proposed semantic of naming: a shift to more business naming of functions over technical